src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,30 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage {
+
+    // Constructor
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    // Locators
+    By userAvatarIcon = By.cssSelector("img[class='avatar']");
+//  By userAvatarIcon = By.cssSelector("img.avatar");
+//  By passwordField = By.cssSelector("input[type='password']");
+//  By submitBtn = By.cssSelector("button[type='submit']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```